app/controllers/cclow/geography/litigation_cases_controller.rb

```diff
@@ -17,6 +17,7 @@ def show
         @sectors = @litigation.laws_sectors.order(:name)
         @keywords = @litigation.keywords.order(:name)
         @responses = @litigation.responses.order(:name)
+        @litigation_sides = @litigation.litigation_sides.map { |ls| LitigationSideDecorator.decorate(ls) }
```

Comment:
We can't use this decorator directly, because this is for the backend site and the link to Geographies points to the admin namespace. We might need to create one specifically for the frontend, like we have on the Laws side.